lib/select.js

```diff
@@ -455,12 +449,7 @@ classBase.Option = React.createClass({
           role: 'button',
           className: this.getClassNames(),
           'data-automation-id': this.props.automationId,
-          tabIndex: -1
-
-          // This is a workaround for a long-standing iOS/React issue with click events.
-          // See https://github.com/facebook/react/issues/134 for more information.
-          , onTouchStart: this.tap,
-
```

Comment:
@niki4810 Hm, this is here from a `src` update in #39. I assume we only update `lib` when a new version is published. Can you undo the `lib` changes?